exporter/awsemfexporter/metric_translator.go

```diff
@@ -56,6 +56,17 @@ type cWMetricStats struct {
 	Sum   float64
 }
 
+// The SampleCount of CloudWatch metrics will be calculated by the sum of the 'Counts' array.
+// The 'Count' field should be same as the sum of the 'Counts' array and will be ignored in CloudWatch.
+type cWMetricHistogram struct {
```

Comment:
I'm curious, is this one defined in the awsemf specification? https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html

Reply:
It is not well described in EMF doc. The Values/Counts defined in [MetricDatum](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_MetricDatum.html) are treated as data points and `SampleCount` is based on it.  The CloudWatch Agent also has [similar logic]( https://github.com/aws/amazon-cloudwatch-agent/blob/main/plugins/outputs/cloudwatch/cloudwatch.go#L547-L554).